<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Êô∫ËÉΩËΩ¶ËæÜË∞ÉÂ∫¶Á≥ªÁªü</string>
  </property>
  <widget class="QPushButton" name="backHomeBut">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ü°∏ËøîÂõû</string>
   </property>
  </widget>
  <widget class="QLabel" name="baseRoadLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string>111</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>1011</width>
     <height>41</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ÈÅìË∑Ø‰ø°ÊÅØÂõæ</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>21</width>
     <height>491</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="carTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Ëµ∑ÁÇπ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ÁªàÁÇπ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Âá∫ÂèëÊó∂Èó¥</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Âà∞ËææÊó∂Èó¥</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ËΩ¶ËæÜ‰ø°ÊÅØ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runBut">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ËøêË°å</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showRuntimeText">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>801</width>
     <height>411</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="carTraceLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>580</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string>111</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>showRuntimeContent(QString)</slot>
 </slots>
</ui>
